fe/fe-core/src/main/java/com/starrocks/connector/iceberg/IcebergMetadata.java

```diff
@@ -52,39 +63,71 @@ public IcebergMetadata(Map<String, String> properties) {
         } else {
             throw new RuntimeException(String.format("Property %s is missing or not supported now.", ICEBERG_CATALOG));
         }
+        tableCache = newCacheBuilder(MAX_TABLE_CACHE_SIZE).build(new CacheLoader<HiveTableName, Table>() {
```

Comment:
nit: HiveTableName? may be we can add other name for this, of course it can be another pr

Reply:
Yes, let's put cod refactor into another PR.